### ISSUE TITLE
UserWarning: Boolean Series key will be reindexed to match DataFrame index.

### ISSUE THREAD

**Issue by yazeed** [freqtrade/freqtrade#2789]
## Step 1: Have you search for this issue before posting it?

Yes

## Step 2: Describe your environment

  * Operating system: MacOS
  * Python Version: Python 3.7.5 (`python -V`)
  * CCXT version: 1.21.76 (`pip freeze | grep ccxt`)
  * Branch: Develop
  * Last Commit ID: 66415d48d4d9d42aaeec6beb1b8fc58dd58cf022 (`git log --format="%H" -n 1`)
  
## Step 3: Describe the problem:

When running the code:

`freqtrade backtesting --strategy Low_BB -i 1h`

It prints out the report just fine except there's an error interrupting these lines:

`
========================================================= SELL REASON STATS =========================================================
/Users/yazeed/Sites/freqtrade/freqtrade/optimize/optimize_reports.py:73: UserWarning: Boolean Series key will be reindexed to match DataFrame index.
  loss = len(result[results['profit_abs'] < 0])
| Sell Reason        |   Count |   Profit |   Loss |   Profit % |
|:-------------------|--------:|---------:|-------:|-----------:|
| trailing_stop_loss |     580 |      350 |    230 |       1.22 |
| stop_loss          |     151 |        0 |    151 |      -1.7  |
`

The error:

`/Users/yazeed/Sites/freqtrade/freqtrade/optimize/optimize_reports.py:73: UserWarning: Boolean Series key will be reindexed to match DataFrame index.`

**Comment by xmatthias** [comment]
It's just a warning - and it's caused by a typo (will be fixed in #2790).

**Comment by hroff-1902** [comment]
@yazeed 
The fix just merged into current develop branch, pls verify the solution.

**Comment by yazeed** [comment]
> @yazeed
> The fix just merged into current develop branch, pls verify the solution.

Just figured, I forgot to reply this one. Fix, verified. :)
